demo/index.html

```diff
@@ -3,13 +3,15 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Auro Web Component Generator | [namespace]-[name] custom element</title>
+    <title>Auro Web Component Generator | auro-button custom element</title>
     <link
       rel="stylesheet"
       type="text/css"
       href="https://unpkg.com/prismjs@latest/themes/prism.css"
     />
-    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <link rel="stylesheet" href="https://unpkg.com/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/@alaskaairux/webcorestylesheets@latest/dist/demoWrapper.css" />
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/@alaskaairux/webcorestylesheets@latest/dist/elementDemoStyles.css" />
```

Comment:
These should be JSDeliver rather than unpkg

Reply:
Addressed.